### DISCUSSION TITLE
Two lines specialization overlaps advisor's name

### DISCUSSION THREAD

**Discussion by elramalho** [joaomlourenco/novathesis#374] [Q&A] [answered]
Hello guys,

I'm writing my master thesis and my specialization is too long, and it takes two lines.
The problem is that overlaps with the name of my advisor.

I tried to break lines (\\\\) where is supposed to break and after the whole name and nothing works. The advisor's name doesn't go down.

How can I solve this?

<img width="300" alt="Screenshot 2022-10-20 at 16 25 20" src="https://user-images.githubusercontent.com/116292986/196991286-14b797e4-4c48-44ce-b7fb-5cf0d06744d0.png">

**Comment by joaomlourenco** [comment]
@elramalho please identify:
Which version of the template are you using?
Which university/school?
In which page(s) are you having troubles with the specialization name?  Can you show the full page(s) and not just a clip?

**Reply by elramalho** [reply]
Sorry, for not giving all the information.
I'm using the latest version of the template (v6.10.10)
I'm studying at University of Minho, School of Engineering
The specialization only appears in the second page of the cover (the one in greyscale)
<img width="536" alt="Screenshot 2022-10-20 at 16 46 24" src="https://user-images.githubusercontent.com/116292986/196996319-79ba26c2-59e2-40a6-b996-5ae9faf2c5cd.png">

**Comment by joaomlourenco** [comment, marked as answer]
@elramalho 
That was a cover “design bug”.
Replace your file `NOVAthesisFiles/Schools/uminho/uminho-defaults.ldf` with this one.  It shall solve the problem.  Please confirm.
[uminho-defaults.zip](https://github.com/joaomlourenco/novathesis/files/9831849/uminho-defaults.zip)

**Reply by elramalho** [reply]
It worked!
Thank you so much and thank you for the quick reply!
<img width="541" alt="Screenshot 2022-10-20 at 17 03 18" src="https://user-images.githubusercontent.com/116292986/197000409-7ed30dc5-6890-4bec-8657-bb664a71ffd2.png">

**Reply by joaomlourenco** [reply]
I'm happy it worked.  This one was easy!  😊
If you find any more issues please report it here.
